src/Covid19Dashboard/index.jsx

```diff
@@ -110,23 +112,52 @@ class Covid19Dashboard extends React.Component {
   render() {
     const { confirmedCount, deathsCount, recoveredCount } = this.getTotalCounts();
 
-    // const displaySeirPlot = Object.keys(this.props.seirObserved).length > 0
-    //   && Object.keys(this.props.seirSimulated).length;
-    // const seirChart = displaySeirPlot ? (<PlotChart
-    //   title='SEIR Model'
-    //   xTitle='Date'
-    //   yTitle='Population Fraction'
-    //   plots={[
-    //     {
-    //       data: this.props.seirObserved,
-    //       name: 'Observed Cases',
-    //     },
-    //     {
-    //       data: this.props.seirSimulated,
-    //       name: 'Simulated Cases',
-    //     },
-    //   ]}
-    // />) : null;
+    const displaySeirPlot = Object.keys(this.props.seirObserved).length > 0 && Object.keys(this.props.seirSimulated).length;
```

Comment:
```suggestion
    const displaySeirPlot = Object.keys(this.props.seirObserved).length > 0 && Object.keys(this.props.seirSimulated).length > 0;
```
that was my mistake, but please fix it for me? 😆